code/src/App.js

```diff
@@ -1,9 +1,75 @@
-import React from 'react'
+import React from "react";
+import { useState, useEffect } from "react";
+import Post from "./component/Post";
+import PostForm from "./component/PostForm";
+
+import Grid from "@material-ui/core/Grid";
 
 export const App = () => {
+  const [apiFetch, setApiFetch] = useState([]);
+  const [postFormValue, setPostFormValue] = useState("");
+
+  const API_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => response.json())
+      .then((json) => setApiFetch(json));
+  }, []);
+
+  // FUNCTIONS
+  const refreshPosts = () => {
+    window.location.reload();
+  };
+
+  const postRequest = () => {
+    const postRequestOption = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ message: postFormValue }),
+    };
+
+    fetch(API_URL, postRequestOption)
+      .then((response) => response.json())
+      .then((data) => setApiFetch([...apiFetch, data]));
+
+    setTimeout(() => refreshPosts(), 500);
+  };
+
+  const handlePostLike = (postID) => {
+    const API_LIKE_URL = `https://happy-thoughts-technigo.herokuapp.com/thoughts/${postID}/like`;
```

Comment:
As this URL is same as the API_URL, just added some at the end, API_URL could've been put in a reusable folder and exported to be reused.